tds/src/main/java/thredds/server/dap4/ThreddsDSP.java

```diff
@@ -1,67 +1,67 @@
-/*
```

Comment:
I understand something may need to be commented out so TDS compiles, my only concern is that this will cause lots of conflicts with @DennisHeimbigner's work. Maybe using /*...*/ to comment it out may help (since then not every line is changed)?

Reply:
I don't think it should cause a conflict, it should just show up as a changed file

Reply:
but I think these will end up being pretty entirely different from how they are now...maybe I'm wrong on that

Reply:
also if there are conflicts I think it's better to have the whole thing commented because then it shows up as one conflicting block, not several conflicts, right?

Reply:
could be? But worst case if there are a lot of conflicts Dennis can make a new branch where he first reverts this change and then rebases his branch onto that, so I don't think it's too big a deal.